core/src/main/resources/swagger/api-docs.yaml

```diff
@@ -6234,3 +6234,4 @@ components:
             email: email authorization
             profile: profile authorization
             https://www.googleapis.com/auth/cloud-billing: cloud billing authorization
+            https://www.googleapis.com/auth/bigquery.readonly: BigQuery read only authorization
```

Comment:
why is this scope needed? I'll read through the rest of the PR and see if the answer is obvious.

Reply:
Rawls doesn't actually need this scope, but WSM does downstream for when we create the uncontrolled resource. WSM does a check to make sure that the user can view the dataset. We were seeing failures in WSM with the default scopes, which was confusing because when we tested with a token from `gcloud auth print-access-token`, it worked. When we inspected the scopes, we saw that the token from `gcloud` had some additional scopes, including the Cloud Platform scope. Cloud Platform is obviously way too broad of a scope to ask for, so we tightened it down to just BigQuery.ReadOnly.

An alternative may be to _not_ require this scope, and instead make the call to WSM as the user's pet, and we can ask Sam to generate the pet token with this scope.

I am slightly leaning towards that solution so we can keep the number of scopes that we request from the user to the minimum possible set.

Reply:
hm, I agree: we shouldn't ask the end user for additional scopes just to add the uncontrolled resource. I would rather solve this at the WSM layer by using a pet or some other solution.

Reply:
we've replaced this with a pet token that has the appropriate scope